packages/block-editor/src/components/link-control/README.md

```diff
@@ -0,0 +1,53 @@
+# Link Control
+
+### className
+
+- Type: `string`
+- Required: NO
+
+### defaultOpen
```

Comment:
On the other PR I noticed `defaultOpen` isn't implemented, so it looks like it can be removed.

Reply:
Yes, documentation is still in progress. :-). Going to update it asap.